File: 1-two-sum/1-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int i, j;
        for(i = 0; i < nums.size(); i++) {
            for(j = 0; j < nums.size(); j++) {
                if(j == i) continue;
                if(nums[j] == target - nums[i]) {
                    return vector<int>{i, j};
                }
            }
        }
        return vector<int>{};
    }
};

File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        table = {}
        for i in range(len(nums)):
            comp = target - nums[i]
            if comp in table:
                print(table)
                return [table[comp], i]
            table[nums[i]] = i